SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDK/SpatialTestRepNotify/SpatialTestRepNotify.cpp

```diff
@@ -151,12 +151,20 @@ void ASpatialTestRepNotify::PrepareTest()
 			}
 
 			// We consciously differ from native UE here
-			if (GetNetDriver()->IsA(USpatialNetDriver::StaticClass()))
+			// Also, the native behaviour changed when going from 4.25 to 4.26.
+			// On older versions, we expect the old array to have 3 elements, but on Spatial and on native starting from 4.26, we expect 2 elements.
+#if ENGINE_MINOR_VERSION >= 26
+			bool bOldArrayShouldHaveTwoElements = true;
```

Comment:
minor: I would prefer to use a const int that defines the number of array elements. Less fragile (no need to magic number the != below) and more expressive.

Reply:
Hmm, though I also do check the additional element when it is available...

Reply:
And this way also lets me include more info in the failure messages...